jest.config.js

```diff
@@ -9,6 +9,11 @@ module.exports = {
   transform: {
     "^.+\\.(ts|tsx)$": "ts-jest",
   },
+  moduleNameMapper: {
```

Comment:
@dipeshwalia  is it the only way to mock png files? should I do that any way?

Reply:
Why we need this ?

Reply:
Why should me merge jest.config into package.json?

Reply:
Because it's mis leading.

React scripts provide us basic jest configuration and not every jest configuration will be entertain by react scripts. 

https://create-react-app.dev/docs/running-tests/#configuration